### ISSUE TITLE
Update web accessibility issues

### ISSUE THREAD

**Issue by hellogreg** [rh-mobb/documentation#446]
This morning, I ran an accessibility crawl with Lumar, which uses axe-core as its engine. It caught a few issues, and it also lead me to manually check and catch a few more.

I'm only listing violations of WCAG 2.1 up through level AA, and not AAA or best practices.

1) The copy icon for code blocks should have 3:1 contrast from its background. This element is used at multiple pages.
![copy icon screenshot](https://github.com/rh-mobb/documentation/assets/642695/7374255e-ab64-46db-8319-222833d80522)

2) A couple pages have empty links:

[Observability](https://rh-cloud-experts.netlify.app/experts/o11y/)
`<a href="/experts/o11y/ocp-grafana/"></a>`

[Installing the Kubernetes Secret Store CSI on OpenShift](https://rh-cloud-experts.netlify.app/experts/misc/secrets-store-csi/)
`<a href="/experts/misc/secrets-store-csi/install-kubernetes-secret-store-driver/"></a>`
`<a href="/experts/misc/secrets-store-csi/uninstall-kubernetes-secret-store-driver/"></a>`

3) Image on the [Azure Front Door with ARO](https://rh-cloud-experts.netlify.app/experts/aro/frontdoor/) page needs `alt` attribute:
![Minecraft screencap](https://github.com/rh-mobb/documentation/assets/642695/142b92dc-07c8-491c-8eca-a96efbed132f)

Also, the word "browser" was mistyped as "broswer" above the image (we've all done it!).

4) Links should not open in new windows unless 1) this is essential to the experience, and 2) the user is tipped off that this will happen. I found a few links that do this, so it could be worth searching all over the site.

For example, there's this link at the [Deploying Grafana on OpenShift 4](https://rh-cloud-experts.netlify.app/experts/o11y/ocp-grafana/) page.
![screencap of link that opens in new window](https://github.com/rh-mobb/documentation/assets/642695/2801f396-9d83-45d7-a54b-38c482dbb68e)

5) Lumar and I also found some issues the [Tutorial: ROSA Prerequisites](https://docs.openshift.com/rosa/rosa_tutorials/rosa-mobb-prerequisites-tutorial.html) page, but that's hosted at docs.openshift.com. I can recheck that page, too, if you ever want to update it. It doesn't have a page language set, there are `<span>` elements as direct children of an `<ol>`, etc. That said, those might be sitewide template issues, so maybe too big to tackle for this project!

**Comment by michaelryanmcneill** [comment]
Thanks @hellogreg for doing this! We will work to address these and get them fixed!

**Comment by michaelryanmcneill** [comment]
- [ ] update icon color contrast
- [ ] Update empty links
- [ ] Add image alt attributes
- [ ] Remove any unnecessary uses of the `target=_blank` attribute on links
